jsettlers.graphics/src/jsettlers/graphics/reader/translator/SettlerTranslator.java

```diff
@@ -44,9 +44,22 @@ public HeaderType getHeaderType() {
 	}
 
 	@Override
-	public short readUntransparentColor(ByteReader reader)
-			throws IOException {
-		return (short) ((reader.read16() << 1) | 0x01);
+	public short readUntransparentColor(ByteReader reader) throws IOException {
+		int read = reader.read16();
+
+		int R5 = (int)((read & 0xF800) >> 11);
```

Comment:
Looks fine, although I prefer lower case variable names for non-constants ;-).

I don't think that the ByteReader is the best place to put that. We could add it as a read-and-convert method. But the way it is now I would rather see it as a static method in one of the image-reading related classes, since the byte reader has nothing to do with the interpretation of images. You can also create a new class (ColorUtils, ...) or simply add it as static method to our Color-Class (which also does 16 Bit to 24 Bit conversion, ...)